src/pages/administration/students/index.vue

```diff
@@ -247,7 +250,9 @@ export default {
 				state.pagination.default || { limit: 10, total: 0 },
 		}),
 		schoolInternallyManaged() {
-			return !this.school.ldapSchoolIdentifier && !this.school.source;
+			return (
+				this.school && !this.school.ldapSchoolIdentifier && !this.school.source
```

Comment:
```suggestion
				!this.school?.ldapSchoolIdentifier && !this.school?.source
```
you can use the new awesome "optional chaining"-syntax here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining